.github/workflows/testpr.yml

```diff
@@ -121,83 +111,77 @@ jobs:
       shell: bash -l {0}
       if: steps.filecheck.outputs.LINUX_YAML_CHANGED == 1 && matrix.os == 'ubuntu-latest' && matrix.platform == 'linux-64'
       run: |
-        git clean -fdx
         cp vinca_linux_64.yaml vinca.yaml
-        vinca --platform linux-64
-        ls
-        cat recipe.yaml
+        mkdir -p recipes
+        $HOME/.pixi/bin/pixi run -e beta -v vinca --platform linux-64 -m -n
+        ls -la recipes
     - name: Generate recipes for linux-aarch64
       shell: bash -l {0}
       if: steps.filecheck.outputs.LINUX_AARCH_YAML_CHANGED == 1 && matrix.os == 'ubuntu-latest' && matrix.platform == 'linux-aarch64'
       run: |
-        git clean -fdx
         cp vinca_linux_aarch64.yaml vinca.yaml
-        vinca --platform linux-aarch64
-        ls
-        cat recipe.yaml
+        mkdir -p recipes
+        $HOME/.pixi/bin/pixi run -e beta -v vinca --platform linux-aarch64 -m -n
+        ls -la recipes
     - name: Generate recipes for osx-64
       shell: bash -l {0}
       if: steps.filecheck.outputs.OSX_YAML_CHANGED == 1 && matrix.os == 'macos-latest' && matrix.platform == 'osx-64'
       run: |
-        git clean -fdx
         cp vinca_osx.yaml vinca.yaml
-        vinca --platform osx-64
-        ls
-        cat recipe.yaml
+        mkdir -p recipes
+        $HOME/.pixi/bin/pixi run -e beta -v vinca --platform osx-64 -m -n
+        ls -la recipes
     - name: Generate recipes for osx-arm64
       shell: bash -l {0}
       if: steps.filecheck.outputs.OSX_ARM_YAML_CHANGED == 1 && matrix.os == 'macos-14' && matrix.platform == 'osx-arm64'
       run: |
-        git clean -fdx
         cp vinca_osx_arm64.yaml vinca.yaml
-        vinca --platform osx-arm64
-        ls
-        cat recipe.yaml
+        mkdir -p recipes
+        $HOME/.pixi/bin/pixi run -e beta -v vinca --platform osx-arm64 -m -n
+        ls -la recipes
     - name: Generate recipes for win-64
       shell: bash -l {0}
       if: steps.filecheck.outputs.WIN_YAML_CHANGED == 1 && matrix.os == 'windows-2019' && matrix.platform == 'win-64'
       run: |
-        echo "CONDA_BLD_PATH=C:\\bld\\" >> $GITHUB_ENV
-        git clean -fdx
         cp vinca_win.yaml vinca.yaml
-        vinca --platform win-64
-        ls
-        cat recipe.yaml
+        mkdir -p recipes
+        $HOME/.pixi/bin/pixi run -e beta -v vinca --platform win-64 -m -n
+        ls -la recipes        cp vinca_win.yaml vinca.yaml
```

Comment:
This looks like a typo

Reply:
Good catch! I copy&pasted this from https://github.com/RoboStack/ros-jazzy/pull/5/files#diff-069a2e75bc939d92ad32a50dada65893abaf603c1a586b97183ec1ca476bba23R149, so we need to remember to fix this also there.

Reply:
Done this just now :)